### PR TITLE
[Storage] Added support for HttpsTrafficOnly

### PR DESCRIPTION
**Pull request by JasonYang-MSFT**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as a link to the swagger spec, used to generate the code.
- [x] The `project.json` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by JasonYang-MSFT** [comment]
Related Swagger PR: https://github.com/Azure/azure-rest-api-specs/pull/1120

**Comment by markcowl** [comment]
@azuresdkci retest this please

File: src/SDKs/Storage/Management.Storage/Microsoft.Azure.Management.Storage.csproj [hunk 1]
```diff
@@ -10,5 +10,6 @@
   </PropertyGroup>
   <PropertyGroup>
     <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+    <Version>6.3.0-preview</Version>
```
Comment (shahabhijeet) [inline src/SDKs/Storage/Management.Storage/Microsoft.Azure.Management.Storage.csproj]: Better option remove VersionPrefix and replace it with <Version>
Add it under top property group where all package meta data is declared.

File: src/SDKs/Storage/Management.Storage/project.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "version": "6.2.0-preview",
+  "version": "6.3.0-preview",
```
Comment (shahabhijeet) [inline src/SDKs/Storage/Management.Storage/project.json]: Delete this file. This file is no longer needed.
Also delete .xproj and global.json (if they are still in the directory)

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

**Comment by JasonYang-MSFT** [comment]
@shahabhijeet Comments are addressed.

**Pull request merged**